include/seqan3/alphabet/detail/member_exposure.hpp

```diff
@@ -94,9 +94,15 @@ struct alphabet_size<alphabet_type_with_members>
  * \returns The letter's value in the alphabet's rank type (usually a `uint*_t`).
  */
 template <typename alphabet_type>
-constexpr underlying_rank_t<alphabet_type> to_rank(alphabet_type const alph)
+//!\cond
     requires requires (alphabet_type alph) { { alph.to_rank() } -> underlying_rank_t<alphabet_type>; }
+//!\endcond
+constexpr underlying_rank_t<alphabet_type> to_rank(alphabet_type const alph) noexcept
+
```

Comment:
remove newline :nail_care: 
same in assign_rank, assign_char and complement below